sdk/Internal/Clock.php

```diff
@@ -12,10 +12,10 @@ public function __construct()
     {
     }
 
-    public function millitime(): string
+    public function zipkinFormattedTime(): int
```

Comment:
I don't know if this is close to a final version, but for me, it's very odd that the Internal Clock "knows" about Zipkin time format.
In my opinion, the internal clock should return time at various resolutions (seconds, milliseconds, nanoseconds) and the ZipkinConverter should be the only one that will know how to convert the timestamps for Zipkin.

Reply:
I agree with you.  I was going to make that change during the hrtime() addition, but I think Levi also had some thoughts.  Letting him share them.